src/scroll-listener.ts

```diff
@@ -4,6 +4,8 @@ import 'rxjs/add/operator/share';
 import 'rxjs/add/observable/empty';
 import { Observable } from 'rxjs/Observable';
 
+import { isWindowExists } from './utils';
```

Comment:
Is there en extra white space line above `import { isWindowExists } from './utils';`? Can you remove it :)